dump-format.mediawiki

```diff
@@ -0,0 +1,66 @@
+<pre>
+  NEP: TBD
+  Title: Unified Neo chain dump format
+  Author: Ekaterina Pavlova <ekt@nspcc.ru>, Roman Khimov <roman@nspcc.ru>
+  Type: Informational
+  Status: Draft
+  Created: 2025-02-14
+</pre>
+
+==Abstract==
+
+This proposal standardizes a single Neo chain dump format that unifies the “full” and “incremental”
+approaches into one consistent structure. It enables different Neo node implementations to remain
+compatible when producing or consuming chain dumps, allowing easy import or export of chain
+data without confusion over file naming or format variations.
+
+==Motivation==
+
+Neo nodes have historically used two chain dump formats: a full dump from block 0 (e.g., chain.acc)
+and an incremental dump from an arbitrary block (e.g., chain.X.acc). This dual approach complicates
+tooling and confuses operators (e.g., chain.0.acc is actually incremental). This NEP unifies both
+formats into one, making dumps interchangeable, simplifying import/restore, and reducing confusion.
+
+==Specification==
+
+===Dump Structure===
+
+==== Dump Header ====
+* '''Start (uint32, LE)''': The first block index included in the dump.
+* '''Count (uint32, LE)''': The total number of blocks contained in the dump.
+
+==== Blocks ====
```

Comment:
I'd expect the format to be somehow similar to https://github.com/neo-project/proposals/blob/master/nep-16.mediawiki#neffile-structure. It's just an idea to keep proposals format unified.